src/main/resources/hudson/plugins/sonar/Messages.properties

```diff
@@ -1,11 +1,14 @@
 SonarPublisher.BadBuildStatus=Skipping sonar analysis due to bad build status {0}
-SonarPublisher.NoInstallation=No SonarQube installation on this job. Available installations count: {0}
-SonarPublisher.NoMatchInstallation=SonarQube installation defined on this job ({0}) do not match anything. Available installations count: {1}.
 SonarPublisher.FixInstalltionTip=If you want to reassign a lot of jobs to a different SonarQube installation see http://docs.sonarqube.org/display/SONAR/Reassign+Jobs+to+Another+SonarQube+Instance
-SonarPublisher.InstallDisabled=SonarQube is disabled ({0}). See global configuration.
+
 SonarPublisher.NoMavenInstallation=No Maven installation found. Call the 'mvn' executable from path
 SonarPublisher.MandatoryProperty=This property is mandatory.
 SonarPublisher.MandatoryPropertySpaces=This property is mandatory and cannot contain spaces.
+
+SonarInstallation.NoMatchInstallation=SonarQube installation defined in this job ({0}) does not match any enabled installation. There are {1} available installations that can be configured.
```

Comment:
What if there is only one installation? The message will look like: There are 1 available installations...